README.md

```diff
@@ -200,6 +201,35 @@ $ git fetch
 $ git checkout v0.3.0
 ~~~
 
+If you've [installed via Homebrew](#homebrew-on-mac-os-x), then upgrade
+via its `brew` command:
+
+~~~ sh
+$ brew update
+$ brew upgrade jawshooah/nodenv/nodenv node-build
+~~~
+
+### Homebrew on Mac OS X
+
+Please note that at this time, the `nodenv` package in Homebrew's main
+repository is *NOT* this package. This package must be installed using the
+instructions below.
+
+As an alternative to installation via GitHub checkout, you can install
+nodenv and [node-build][] using the [Homebrew](http://brew.sh) package
+manager on Mac OS X:
+
+~~~
+$ brew update
+$ brew tap jawshooah/nodenv
+$ brew install jawshooah/nodenv/nodenv
+$ brew install --HEAD node-build
```

Comment:
I requested an initial release at OiNutter/node-build#32 a while ago, once that's done I can update this PR.